string effector_name
--------------------------------
float32[] position
